drivers/gles2/rasterizer_gles2.h

```diff
@@ -49,6 +49,7 @@
 #ifndef GLES2_INCLUDE_H
 #include <GLES2/gl2.h>
 #else
+// cppcheck-suppress *
```

Comment:
This is another false-positive. cppcheck throws an error about no header in #include statement on the following line. However this error message is not documented so I do not know the correct error ID to use to disable it.